promgen/models.py

```diff
@@ -370,6 +370,24 @@ def __str__(self):
         return "{}:{}{}".format(self.job, self.port, self.path or "/metrics")
 
 
+class ExporterLabel(models.Model):
+    name = models.CharField(
+        blank=False,
+        max_length=128,
+    )
+    value = models.CharField(
+        max_length=128,
+    )
+    is_parameter = models.BooleanField(default=False)
```

Comment:
as mentioned above, I think having a toggle for this is going to be more confusing to the user, than doing a bit of filtering on `key.startswith('__param')`